integration-tests/src/test/resources/performance-features/cloud-events-performance.feature

```diff
@@ -42,17 +42,20 @@ Feature: Sending cloud events performance tests
             maxDuration: 6m
             scenario:
             - send-cloud-event:
+              - timestamp:
+                  toVar: submitted_at
               - httpRequest:
                   POST: ${bridge.my-perf-bridge.endpoint.path}
                   body: |
                     {
                       "specversion": "1.0",
                       "type": "webhook.site.invoked",
                       "source": "WebhookActionTestService",
-                      "id": "webhook-test",
+                      "id": "webhook-perf-test",
                       "data": {
                           "bridgeId": "${bridge.my-perf-bridge.id}",
-                          "message": "hello bridge"
+                          "message": "hello bridge",
+                          "submitted_at": "${submitted_at}"
```

Comment:
You can reuse `time` property in cloud event for this - https://github.com/cloudevents/spec/blob/v1.0/spec.md#time